### DISCUSSION TITLE
Plugin folder location Mac

### DISCUSSION THREAD

**Discussion by NotDari** [elgatosf/elgato#18] [Help]
![image](https://github.com/elgatosf/elgato/assets/87239935/83bf499d-18a2-457f-a953-65bec9fb3aa6)

Hiya, according to the website, plugins can be found at this location, but a com.elgato.StreamDeck folder doesn't exist in Application Support. Where can I find the plugin location?

Thanks!

**Comment by tiptronic** [comment]
It is there... just open Terminal.app and past the next line into it:

```bash
open ~/Library/Application\ Support/com.elgato.StreamDeck/Plugins
```

or if you prefer without escaping the path:

```bash
open ~"/Library/Application Support/com.elgato.StreamDeck/Plugins"
```

or (replace '>yourusername<' with your user's name)

```bash
open "/Users/>yourusername</Library/Application Support/com.elgato.StreamDeck/Plugins"
```

**Comment by NotDari** [comment]
Apologies, I was looking at the wrong Application Support. Stupid mistake! Thank you for your help
